### ISSUE TITLE
[Windows] README config example not working because of quotes

### ISSUE THREAD

**Issue by yuriy636** [pepershukov/ytm-yld#2]
First of all, thanks for this tool ❤

I couldn't make it work following the example:

```ini
[ytm-yld]
cookie = 'C:\Users\yuriy\Downloads\music.youtube.com_cookies.txt'
mode = 'd'
json = 'C:\Users\yuriy\Music\ytm-yld\metadata.json'
songs-json = 'C:\Users\yuriy\Music\ytm-yld\songs_metadata.json'
output = 'C:\Users\yuriy\Music\ytm-yld'
```

Was getting errors like:

> [2023-05-27 09:43:14.523516] [ERROR] [main 205] Invalid or non-existant [output] folder.
> [2023-05-27 09:43:14.694533] [INPUT] [main 206] Enter the absolute path to an [output] folder:

Then I tried removing the quotes from the paths:

```ini
[ytm-yld]
cookie = C:\Users\yuriy\Downloads\music.youtube.com_cookies.txt
mode = d
json = C:\Users\yuriy\Music\ytm-yld\metadata.json
songs-json = C:\Users\yuriy\Music\ytm-yld\songs_metadata.json
output = C:\Users\yuriy\Music\ytm-yld
```

And it worked correctly.

**Comment by pepershukov** [comment]
Thank you for your feedback, @yuriy636 !

I have edited the README to not include quotations, because indeed you do not need to put them there.

While testing myself regarding the issue, I also encountered that parameters of `no-[type]` *(music metadata options)* were being parsed poorly. This is relevant because they are all parsed as strings, and because they are strings, unlike a list I was intending for it to be, this error occurs: `TypeError: 'in <string>' requires string as left operand, not bool`.

I will hopefully push a hotfix some time on the following week regarding this issue. Until then, I will remove the notations for these options completely. Thanks again for bringing my attention to this.
